www/js/app.js

```diff
@@ -75,7 +75,9 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
 
     // Set parameters and associated UI elements from cookie
     params['hideActiveContentWarning'] = cookies.getItem('hideActiveContentWarning') === 'true';
+    params['showUIAnimations'] = cookies.getItem('showUIAnimations') === 'true';
```

Comment:
On first use (or, more generally, when the cookie is not set at all), this disables the animation feature.
My opinion is that this feature should be on by default (we still let the user disable it if he/she wants).

So, this line should be modified, so that, if the cookie does not exist, we set `params['showUIAnimations']` to true

Reply:
@mossroy 
Please review the changes I made in the recent commit.
I have implemented as suggested .